src/package.xml

```diff
@@ -2691,6 +2691,34 @@
         <members>Trigger_Handler__c-iw</members>
         <members>Trigger_Handler__c-ja</members>
         <members>Trigger_Handler__c-nl_NL</members>
+        <members>Data_Import_Field_Mapping_Set__mdt-de</members>
```

Comment:
@patrick-yan-sf  Were these not in the package.xml in the base PR?

Reply:
@ptewson-sfdo

Reply:
Nope - I apparently missed that step, although I thought they had been in there. They are there here now!